From b301f0a21e231a725dcc66d46e935f71dd2c4f6f Mon Sep 17 00:00:00 2001
From: Derek Higgins <derekh@redhat.com>
Date: Wed, 10 Jul 2013 14:28:06 +0100
Subject: [PATCH] Ensure /var/log/heat/engine.log is owned by heat

If heat-manage db_sync creates the logfile before the
heat engine service then it will be owned by root causing
heat-engine to error.

Fixes bug 1199789

Change-Id: Ib23a0d46ebf354a3293a64215715038497faf5b0
---
 elements/boot-stack/bin/reset-db |    1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/boot-stack/bin/reset-db b/elements/boot-stack/bin/reset-db
index d903614..52186fc 100755
--- a/elements/boot-stack/bin/reset-db
+++ b/elements/boot-stack/bin/reset-db
@@ -27,6 +27,7 @@ $venvs/glance/bin/glance-manage db_sync
 
 os-db-create heat heat $db_pass
 $venvs/heat/bin/heat-manage db_sync
+chown heat:heat /var/log/heat/engine.log
 
 os-db-create ovs_neutron neutron $db_pass
 
-- 
1.7.9.5

